.github/workflows/test-openmcyclus.yml

```diff
@@ -11,26 +11,37 @@ jobs:
       - name: Setup Conda
         uses: conda-incubator/setup-miniconda@v2
         with:
-          miniforge-variant: Mambaforge
           miniforge-version: latest
           activate-environment: openmcyclus-env
-          environment-file: environment.yml
-          use-mamba: true  
 
       - name: Conda config
         run: |
           conda config --env --set pip_interop_enabled True 
-      
-      - name: Update Environment
-        run: |
-          mamba env update -n openmcyclus-env -f environment.yml
 
       - name: Install pytest
         run: conda install -y pytest
 
       - name: Build OpenMC 
         run: |
-          mamba install openmc
+          conda install -y python=3.7 cyclus cycamore 
+          HDF5_DIR=$CONDA_PREFIX && pip install h5py
+   
+      - name: OpenMC dependencies
+        run: |
+          sudo apt -y update
+          sudo apt install -y libhdf5-dev
+
+      - name: Download OpenMC
+        uses: actions/checkout@v3
+        with: 
+          repository: openmc-dev/openmc
+          path: openmc
+          submodules: recursive
+
+      - name: Build OpenMC from source
+        run: |
+          run: build-openmc.sh
+
 
       - name: Install OpenMC cross section library
         run: echo "OPENMC_CROSS_SECTIONS=cross_sections.xml" >> $GITHUB_ENV
```

Comment:
It looks like right now there is a syntax error in the ci scripts, so I will fix that. But because of a seg fault when importing OpenMC into a cyclus script (See issue #12), there isn't much point in running the integration tests in a CI environment, because they will all fail due to the seg fault. I can make sure that the environment builds and the unit tests pass, but that's the best I can do until Issue #12 is resolved.

Reply:
Oh yes sorry for not reading your PR comments properly

Reply:
All good. It's important to have the CI as functional as possible. I'll fix the syntax error int he CI on this branch, but do the bulk of the CI changes/maintenance on another branch to keep this PR from getting too big.